### PR TITLE
diff: refine log

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 

fix issue https://github.com/pingcap/tidb-tools/issues/360 https://github.com/pingcap/tidb-tools/issues/361

### What is changed and how it works?

1. print basic information at the beginning, include git hash, branch and so on
2. print config before check
3. print debug log for slow SQL, and increase the threshold value for slow query.

### Check List 

Tests 

 - Manual test (run diff and check the log)

### CODE REVIEW

File: pkg/dbutil/common.go [hunk 1]
```diff
@@ -648,14 +650,14 @@ func ExecSQLWithRetry(ctx context.Context, db *sql.DB, sql string, args ...inter
 		_, err = db.ExecContext(ctx, sql, args...)
 		takeDuration := time.Since(startTime)
 		if takeDuration > SlowWarnLog {
-			log.Warn("exec sql slow", zap.String("sql", sql), zap.Reflect("args", args), zap.Duration("take", takeDuration))
+			log.Debug("exec sql slow", zap.String("sql", sql), zap.Reflect("args", args), zap.Duration("take", takeDuration))
```
Comment (csuzhangxc) [inline pkg/dbutil/common.go:653]: No `Warn` for `SlowWarnLog` now, how about giving `SlowWarnLog` another name?

Comment (WangXiangUSTC) [reply pkg/dbutil/common.go:653]: update in 758aca3

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Pull request merged**